modules/local/fetchtool_reads.nf

```diff
@@ -10,22 +10,29 @@ process FETCHTOOL_READS {
     path fetchtool_config
 
     output:
-    tuple val(meta), path("download_folder/${study_accession}/raw/${reads_accession}*.fastq.gz"), emit: reads
-    path "versions.yml"                                                                         , emit: versions
+    tuple val(meta), path("download_folder/${study_accession}/raw/${reads_accession}*.fastq.gz"), env(library_strategy), env(library_layout), emit: reads
+    // The '_mqc.' is for multiQC
+    tuple val(meta), path("download_folder/${study_accession}/fetch_tool_mqc.tsv")                                     , emit: metadata_tsv
+    path "versions.yml"                                                                                                , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "$reads_accession"
+    def prefix = task.ext.prefix ?: "${reads_accession}"
     def private_study = params.private_study ? "--private" : ""
     """
     fetch-read-tool -d download_folder/ \\
-    -p $study_accession \\
-    -ru $reads_accession \\
-    -c $fetchtool_config \\
-    -v $private_study $args
+    -p ${study_accession} \\
+    -ru ${reads_accession} \\
+    -c ${fetchtool_config} \\
+    -v ${private_study} ${args}
+
+    library_strategy=\$(grep ${reads_accession} download_folder/${study_accession}/${study_accession}.txt | cut -f 7)
+    library_layout=\$(grep ${reads_accession} download_folder/${study_accession}/${study_accession}.txt | cut -f 5)
+
+    cp download_folder/${study_accession}/${study_accession}.txt download_folder/${study_accession}/fetch_tool_mqc.tsv
```

Comment:
mv? :)

Reply:
I will remove the `_mqc` file, I couldn't make MultiQC to accept it withut that prefix